Bootstrap/Podfile.lock

```diff
@@ -35,10 +34,10 @@ CHECKOUT OPTIONS:
     :git: https://github.com/facebook/ios-snapshot-test-case.git
 
 SPEC CHECKSUMS:
-  FBSnapshotTestCase: 094f9f314decbabe373b87cc339bea235a63e07a
+  FBSnapshotTestCase: '094f9f314decbabe373b87cc339bea235a63e07a'
```

Comment:
We talked about it this week:

<img width="565" alt="screen shot 2017-05-26 at 11 23 56" src="https://cloud.githubusercontent.com/assets/49038/26491063/ddc77078-4205-11e7-8e56-90769002c356.png">